templates/BLUE.html

```diff
@@ -33,7 +33,11 @@
             <p>Administrative</p>
         </div>
         <a onclick="display_errors({{ errors }})">Startup errors</a>
-        <a href="/docs/index.html" target="_blank">Docs</a>
+        {% if plugins | selectattr('name', 'eq', 'fieldmanual') | list %}
```

Comment:
I think I would remove this from red.html and blue.html, as we can retrain people to look at plugins -> field manual for their docs

Reply:
Sure, makes sense. 3c305ab14e55e20b9f36cfb6ebb2d3cd0b6cdace